_episodes/05-accelerating-lammps.md

```diff
@@ -34,46 +34,20 @@ But to obtain the Kokkos accelartion, LAMMPS needs to be built using Kokkos for
 
 The list of LAMMPS features that is supported by Kokkos is given below:
```

Comment:
If we can just provide a link here do so, this list is a moving target, we should use a link so our information is up to date.